recipes/clingo-dl/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake .. \
```

Comment:
```suggestion
cmake ${CMAKE_ARGS} .. \
```

Reply:
Thanks, I did not know about this variable.